doc/tang-nagios.1.adoc

```diff
@@ -1,11 +1,16 @@
-tang-nagios(1) -- A Nagios plugin for Tang
-==========================================
+tang-nagios(1)
+==============
+:doctype: manpage
 
-## SYNOPSIS
+== NAME
 
-`tang` -u URL
+tang-nagios - A Nagios plugin for Tang
```

Comment:
This command name is incorrect. This program is installed a nagios plugin named `tang`.

Reply:
See https://github.com/latchset/clevis/pull/65#discussion_r202285503